src/test/regress/sql/multi_create_insert_proxy.sql

```diff
@@ -65,7 +65,7 @@ COPY pg_temp.:"proxy_tablename" FROM stdin;
 4	sed do eiusmod
 5	tempor incididunt ut
 6	labore et dolore
-7	\N
```

Comment:
This changes the meaning of the test. Previously we sent a `NULL`, which would cause the `INSERT` to fail. This change now fails at the `COPY` layer, since a row is entirely missing a column value.

What error do we get if this is left alone?